README.md

```diff
@@ -267,6 +267,42 @@ console.log(RNLocalize.uses24HourClock());
 
 ---
 
+### isAutoDateAndTime()
```

Comment:
Could you rename the functions `usesAutoTimeAndDate` and `usesAutoTimeZone ` to keep consistency with `uses24HourClock` or `usesMetricSystem`?

Reply:
absolutely

Reply:
done